### DISCUSSION TITLE
Q: Simple `<c-n>` mapping

### DISCUSSION THREAD

**Discussion by barrettruth** [saghen/blink.cmp#598] [General]
I'd like to bind `<c-n>` to:

- If the completion menu is not open, open it
- Select the next item

For the life of me I can't get this config to work using builtin mappings. `<c-space>` always shows the correct completions (i.e. `vim.api` types), but `<c-n>` only shows buffer-related ones. Any help would be appreciated.

For example, after exiting out of the completion menu with `<c-e>`, `<c-space>` yields:

![image](https://github.com/user-attachments/assets/c437716e-1278-404f-ad7d-434f1c7537a0)

While `<c-n>` yields:

![image](https://github.com/user-attachments/assets/ce71e269-95ca-461b-8bfd-03dd46123d7b)

**Comment by saghen** [comment]
It should work with `['<C-n>'] = { 'show', 'select_next' }`. What's your current keymap config?

**Comment by barrettruth** [comment]
#599
